recipe/meta.yaml

```diff
@@ -22,15 +25,8 @@ build:
     - /usr/lib/libcharset.1.dylib  # [osx]
 
 requirements:
-  build:
-    - pkg-config  # [osx and not arm64]
-    - stack  # [not aarch64 and not arm64]
   host:
-    - zlib  # [linux and not aarch64]
-    - zlib  # [osx and not arm64]
-  run:
-    - gmp  # [linux and not aarch64]
-    - zlib  # [linux and not aarch64]
+    - zlib
```

Comment:
I don't think we need `zlib` in general on host and it was not required before I tried to build it on source.

Reply:
Good spot, thanks!